input/fsh/SD_Extensions.fsh

```diff
@@ -21,18 +16,27 @@ Device instances are not specified here. Those are represented by resources of t
 // http://hl7.org/fhir/us/mcode/2021May/StructureDefinition-mcode-radiotherapy-fractions-delivered.html
 Extension: RadiotherapyFractionsPrescribed
 Id: codexrt-radiotherapy-fractions-prescribed
-Title: "(Future) Number of Prescribed Fractions"
+Title: "Number of Prescribed Fractions"
 Description: "The number of prescribed Fractions in this scope.
 This extension SHALL only be present if the treatment is structured as countable Fractions, for example in a Phase Prescription."
 * . ^short = "Number of Prescribed Fractions"
 * value[x] ^short = "Number of Prescribed Fractions"
 * value[x] only positiveInt
```

Comment:
General comment for all values:   They should be required 1..1.    The extension isn't required, but if you include the extension the value should be required.   See my variant of your branch with -sak suffix.

Reply:
Agreed. All 1..1.